src/main/kotlin/no/nav/familie/ks/sak/api/BisysController.kt

```diff
@@ -21,7 +20,7 @@ import org.springframework.web.bind.annotation.RestController
 @RestController
 @RequestMapping("/api/bisys")
 @ProtectedWithClaims(issuer = "azuread")
-class BisysController(private val bisysService: BisysService, private val tilgangService: TilgangService) {
```

Comment:
Burde vi ikke bruke tilgangsservice her? Hva skjer om en saksbehandler kaller dette endepunktet på en kode 6 sak?

Reply:
Se https://github.com/navikt/familie-ks-sak/blob/70a3cc789bf050b30a8528f716c61272b0cb63a7/src/main/kotlin/no/nav/familie/ks/sak/config/BisysConfig.kt#L4, det brukes et filter istedenfor for bisys.